### PR TITLE
update various packages

### LINKED ISSUE #99: Bump versions
A lot of outdated packages, at least according to pyproject.toml
image

### PR DESCRIPTION
**Pull request by MRuecklCC**
- [x] Requires merge #104 first
- [x] Requires rebase on dev after merge of #104
- Resolves #99

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -15,36 +15,36 @@ meta-lookup = 'main:main'
 
 [tool.poetry.dependencies]
 python = "^3.9.1"
```
Comment (MRuecklCC) [inline pyproject.toml:17]: I dropped all the patch versions and updated the lock file, so we will use the latest patch for all dependencies.

File: pyproject.toml [hunk 2]
```diff
@@ -15,36 +15,36 @@ meta-lookup = 'main:main'
...
-pytest-asyncio = "^0.18.3"
-locust = "^2.9.0"
-httpx = "^0.23.0"
+pylint = "^2.14"
+pre-commit = "^2.19"
+pytest = "^6.2"
+pytest-cov = "^2.12"
+pytest-mock = "^3.7"
+pandas = "^1.4"
+altair = "^4.2"
```
Comment (RMeissnerCC) [inline pyproject.toml:38]: Nice - but I cannot find a usage of altair anymore. Could you check, please, if they can be deleted?

**Review by RMeissnerCC — APPROVED** [review]
Nice - could you check, if all dependencies are still in use? A quick global search would be fine for me. E.g. Altair is nowhere used.

**Comment by MRuecklCC** [comment]
Removed `altair`, `altair-viewer`, and `wily` dev depdendencies:

image

File: pyproject.toml (hunk 2 above)
Comment (MRuecklCC) [reply pyproject.toml]: removed dev dependencies:
 - altair
 - altair-viewer
 - pandas
 - wily
 - vulture
 - pip-licenses
 - pyinstrument

**Comment by MRuecklCC** [comment]
Not sure why the pipeline is failing, there are some 503 errors coming out of the poetry / pip setup and restoring the cache. Eventually it's just a glitch with github - Will recheck in the afternoon or tomorrow.

**Pull request merged**